site/en/docs/webstore/images/index.md

```diff
@@ -19,14 +19,14 @@ browser:
 - [Promotional images][4]
 - [Screenshots][5]
 
-Only the app icon, a small promotional image, and a screenshot are mandatory. However, providing
-attractive versions of both required and optional images increases your app's chances of getting
-noticed. For example, your app can't be featured unless you provide a large or marquee promotional
+Only the extension icon, a small promotional image, and a screenshot are mandatory. However, providing
+attractive versions of both required and optional images increases your extension's chances of getting
+noticed. For example, your extension can't be featured unless you provide a large or marquee promotional
 image.
```

Comment:
For future consideration, I think it may be best if we expand on merchandizing requirements in a dedicated page and cross link to it from here.

Reply:
SGTM, noted.